src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="empty"></div>
```

Comment:
Each star in a block should have a class stars__star and no extra classes or inline styles